### PR TITLE
Create béchamel-sauce.md

### PR DESCRIPTION
**Pull request by yiusa**
fixed the file extension

### CODE REVIEW

**Comment by sylGauthier** [comment]
Rename in `bechamel-sauce.md`. File names should be in lower cap, in ASCII characters. Also, the ordering on the front page is alphabetical by file name, people expect to find Béchamel at B, not at C.

File: src/Classic-Béchamel-sauce.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Contribution
+
+yiusa
+
+
+
+ - eth: `0x68F1317c6512f0267fA711cafB6C134Ae968FA80`
+
+
+
```
Comment (hallzy) [inline src/Classic-Béchamel-sauce.md:36]:
```suggestion
yiusa
 - eth: `0x68F1317c6512f0267fA711cafB6C134Ae968FA80`

```

**Comment by yiusa** [comment]
> Rename in `bechamel-sauce.md`. File names should be in lower cap, in ASCII characters. Also, the ordering on the front page is alphabetical by file name, people expect to find Béchamel at B, not at C.

thanks for the suggestion, im new to git so im learning as i go. a friend linked me this because he knows i hate the way recipe sites are setup nowadays and thought i'd contribute.

**Comment by pahlse** [comment]
I think you'r supposed to rename in ASCII characters. `bechamel-sauce.md` instead of `béchamel-sauce.md`.

**Pull request closed without merging**